README.md

```diff
@@ -59,7 +69,8 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+The first line (var weather;) assigns a variable weather, but doesn't assign it a value - it's undefined. The second line (weather = "sunny";) is setting that variable weather a value - so now the variable weather contains a string "sunny". The third line (weather === "sunny") checks if weather is equivalent to "sunny" - if it is, the expression will evaluate to true, and if it is not, the expression will evaluate to false. Since the variable weather does in fact contain the string "sunny", the expression evaluates to true.
```

Comment:
All of this is true, but the question specifically asked what each line _evaluated to_. 

Line 1: `var weather;` evaluates to `undefined`.
Line 2: `weather = "sunny"` evaluates to `"sunny"`, because the `=` operator will always evaluate to the value that it's assigning.
Line 3: `weather === "sunny"` evaluates, as you indicated, to `true`.